ci-operator/step-registry/openshift-qe/ocpbugs-76596-ct-flush/openshift-qe-ocpbugs-76596-ct-flush-commands.sh

```
@@ -0,0 +1,350 @@
+      - name: c
+        image: gcr.io/google_containers/pause:3.1
+        resources:
+          requests:
+            cpu: 1m
+            memory: 4Mi
+BEOF
+
+echo "Waiting for pods to be Running..."
+for i in $(seq 1 12); do
+  sleep 30
+  RUNNING=$(oc get pods -n burst-test --no-headers 2>/dev/null | grep Running | wc -l)
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)